packages/toolpad-app/src/server/index.ts

```diff
@@ -315,6 +316,16 @@ async function createToolpadHandler({
   const publicPath = path.resolve(currentDirectory, '../../public');
   router.use(express.static(publicPath, { index: false }));
 
+  const childrenPublic = path.resolve(project.getRoot(), './public');
+  const sameNameStatic = await compareSameLayerFolder(publicPath, childrenPublic);
```

Comment:
This is not necessary.
Over time we will add a warning when there is a _Toolpad page_ overlap with the public folder. a first version does not need this warning as I'm concerned

The `publicPath` thing at the moment is mostly there because it hosts the sample movies.json file we use as a default in the REST queries. I want to remove it over time. No need to add warnings there.

Reply:
>public folder takes precedence. We can add a warning in dev mode if we detect a collision

doesn’t the first version care the warning ?
could we leave the code of `compareSameLayerFolder`? I have added  a lot of the tests 😂

Reply:
I just want to confirm, can I simply delete this comparison code?
`const sameNameStatic = await compareSameLayerFolder(publicPath, childrenPublic);`

Reply:
> doesn’t the first version care the warning ?

It wouldn't catch a collision with a Toolpad page. It would only catch collisions with the contents of https://github.com/mui/mui-toolpad/tree/master/packages/toolpad-app/public Which we should probably remove because it contains files relevant to the Toolpad editor, not the Toolpad app

Reply:
> > doesn’t the first version care the warning ?
> 
> It wouldn't catch a collision with a Toolpad page. It would only catch collisions with the contents of https://github.com/mui/mui-toolpad/tree/master/packages/toolpad-app/public Which we should try remove in the future because it contains files relevant to the Toolpad editor, not the Toolpad app

Which kind of page(path) does it belong to a `Toolpad page` ？ 
Sorry I have a little bit confused.

Reply:
"There is only this link for user access. Would you like to inspect whether there is a collision with 'http://localhost:3000/_toolpad/app/pages/page'?"

Reply:
Yes, but as I said before, it can be a separate effort. Better focus on creating docs for this feature.

Reply:
> Yes, but as I said before, I prefer it to be a separate effort. I's better to focus on creating docs for this feature.

![image](https://github.com/mui/mui-toolpad/assets/27261522/54bb149f-a2bc-4bc9-b9eb-8ac282f28bf8)

>"There is only this link for user access. Would you like to inspect whether there is a collision with 'http://localhost:3000/_toolpad/app/pages/page'?"

I have done this